### PR TITLE
BP-555: Add background hover mixin and docs

### PR DESCRIPTION
**Pull request by jovana-marceta**
https://jira.enigmatry.com/browse/BP-555

### CODE REVIEW

File: projects/scss-foundation/docs/background-hover.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Background hover method
+
+Hover utility for easily define the regular and hover states of background.
+
+## Background hover
+
+@use scss-foundation/src/states/hover;
```
Comment (typhoon41) [inline projects/scss-foundation/docs/background-hover.md:7]: Actually this is inside modules? Link is incorrect then.

File: projects/scss-foundation/docs/background-hover.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Background hover method
+
+Hover utility for easily define the regular and hover states of background.
```
Comment (typhoon41) [inline projects/scss-foundation/docs/background-hover.md:3]: Make this description bit flexible by adding plural from start; in other words:
"Hover utilities for easy definition of regular and hover background states.

File: projects/scss-foundation/docs/background-hover.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Background hover method
```
Comment (typhoon41) [inline projects/scss-foundation/docs/background-hover.md:1]: How do we navigate to this file from main documentation file?

**Review by typhoon41 — CHANGES_REQUESTED** [review]

File: projects/scss-foundation/docs/background-hover.md (hunk 2 above)
Comment (jovana-marceta) [reply projects/scss-foundation/docs/background-hover.md]: I noticed that the paths were incorrect in other doc files as well, so I fixed them.

Comment (typhoon41) [reply projects/scss-foundation/docs/background-hover.md]: That's great unless @Cvetanovic has done it already on other branch :)

**Review by typhoon41 — APPROVED** [review]

**Pull request merged**